src/Mailgun/Api/Suppressions/Complaint.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Api;
+
+use Mailgun\Assert;
+use Mailgun\Resource\Api\Suppressions\Complaint\CreateResponse;
+use Mailgun\Resource\Api\Suppressions\Complaint\DeleteResponse;
+use Mailgun\Resource\Api\Suppressions\Complaint\IndexResponse;
+use Mailgun\Resource\Api\Suppressions\Complaint\ShowResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class Complaint extends HttpApi
+{
+    use Pagination;
+
+    /**
+     * @return string
+     */
+    protected function getPaginationBase()
```

Comment:
Yeah, this needs to be gone now that I've removed that method from the Pagination trait.